### PR TITLE
Add custom elements manifest analyzer

### PR DESCRIPTION
**Pull request by luwes**
Related #388 

This change is the start for adding a custom elements manifest.
Related to https://github.com/muxinc/media-chrome/discussions/158

A little downside of Media Chrome using constant variables is that the analyzer can't pick the attributes and events up automatically. An alternative is to use special JSDoc syntax to get these picked up.
https://custom-elements-manifest.open-wc.org/analyzer/getting-started/#supported-jsdoc

I added one example for the Airplay button.

~The custom function `defineCustomElement()` in Media Chrome was causing issues too, luckily this could be automated with a plugin.~ The analyzer expects `customElements.define('my-el')`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +19,7 @@
     "build:cjs": "esbuild src/js/*.js src/js/*/*.js src/js/*/*/*.js --target=es2019 --format=cjs --outdir=dist/cjs",
     "build:iife": "esbuild src/js/index.js --bundle --target=es2019 --format=iife --outdir=dist/iife --minify --sourcemap --global-name=MediaChrome && yarn size",
     "build:react": "node ./scripts/react/build.js",
+    "build:manifest": "cem analyze --config scripts/custom-elements-manifest.config.js",
     "build": "yarn build:esm && yarn build:cjs && yarn build:iife && yarn build:react && yarn build:types",
```
Comment (cjpillsbury) [inline package.json:23]: suggestion(non-blocking): Update `build` script to include `build:manifest`

**Review by cjpillsbury — COMMENTED** [review]
todo(blocking): Add the manifest to our .gitignore

**Review by cjpillsbury — CHANGES_REQUESTED** [review]
This is awesome. Can't wait to get it added to all of our elements! 🎉 Added one small todo and one non-blocking suggestion so we automatically build the manifest on releases. Otherwise, I'm lovin' it.

File: package.json (hunk 1 above)
Comment (luwes) [reply package.json:23]: will do

**Comment by dylanjha** [comment]
> Otherwise, I'm lovin' it.

Me too. Very happy to see this getting movement!

**Comment by luwes** [comment]
> todo(blocking): Add the manifest to our .gitignore

@cjpillsbury it's output in the `dist` folder 😉

**Review by cjpillsbury — APPROVED** [review]
Approved assuming the docs build/deploy fail is unrelated. 🎉

**Comment by heff** [comment]
> A little downside of Media Chrome using constant variables is that the analyzer can't pick the attributes and events up automatically.

Should we not use constants in the places where the analyzer doesn't want them? It's not like those strings are gonna change.

**Pull request merged**